pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/ViewFilter.java

```diff
@@ -120,42 +121,30 @@ public ViewFilter(String filterName, Map<String, List<Object>> columns, String i
         this.realmId = realmId;
     }
 
-    private static SimpleResult checkUniqueFilterName(String suggestedName) {
```

Comment:
I reworked this a little bit to clarify what the method does, and returning a boolean instead of the more opaque `SimpleResult`.  Since there's only one caller and the method is used to check for uniqueness, there was an opportunity to simplify this a bit.